packages/types/src/story/page.ts

```diff
@@ -35,4 +46,9 @@ export interface Page {
   backgroundColor: Pattern;
   type: 'page';
   groups: Groups;
+  id: string;
+  pageAttachment?: PageAttachment;
+  advancement: Advancement;
+  backgroundAudio: BackgroundAudio;
+  shoppingAttachment: ShoppingAttachmentType;
```

Comment:
All of these are optional, no?

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12382/commits/a1cea5e2504a0731e448b3e5895b18eb6562016d